src/objects/OcclusionQueryMesh.js

```diff
@@ -0,0 +1,40 @@
+import { Mesh } from './Mesh.js';
+
+/**
+ * @author simon paris / https://github.com/simon-paris
+ */
+
+function OcclusionQueryMesh() {
```

Comment:
If you introduce a new object type, it's necessary to adjust `ObjectLoader` and `Object3D.toJSON()` so it's possible to serialize/deserialize a scene with objects of type `OcclusionQueryMesh`.

Reply:
Since OcclusionQueryMesh has a callback function property, and also needs to be disposed properly, would it be acceptable to leave serialization unsupported and just print a warning?

Reply:
Um, I see what you mean. Let me think about this...